x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -6,13 +6,12 @@
 import fs from 'fs';
 import os from 'os';
 import path from 'path';
-// @ts-ignore
-import puppeteer from 'puppeteer-core';
 import rimraf from 'rimraf';
 import * as Rx from 'rxjs';
 import { map, share, mergeMap, filter, partition, ignoreElements, tap } from 'rxjs/operators';
 import { InnerSubscriber } from 'rxjs/internal/InnerSubscriber';
 
+import { launch, Browser, Page } from '../puppeteer';
```

Comment:
We can also do 
```
import { Browser, Page, LaunchOptions } from 'puppeteer';
```

And when we call `launch` we can do:
```
        browser = await launch({
          pipe: !this.browserConfig.inspect,
          userDataDir,
          executablePath: this.binaryPath,
          ignoreHTTPSErrors: true,
          args: chromiumArgs,
          env: {
            TZ: browserTimezone,
          },
        } as LaunchOptions);
```

That will ensure that we are doing a good job at building the object literal that gets passed. We wouldn't need the `puppeteer` file, but on the other hand, the `launch` method would not be typed quite as well as you have it.